assignments/prototypes.js

```diff
@@ -102,7 +138,7 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+
```

Comment:
Fantastic job, Daisy, seriously! I know you were a little overwhelmed by this project, but you calmed yourself down, went at your own pace, and you killed it :D This is exactly what we were looking for, and if you can do the same thing on your Sprint Challenge tomorrow, you are going to be just fine!